lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -157,6 +157,12 @@ def __init__(self, *args):
 ]
 ```
 
+It is **important** to note that the `job_name` must be a static
+hardcoded string. For instance **do not** set the job name using your
+charm's `unit.name`. If the job name changes on upgrading your charm,
+scaling the number of units etc., this will break the continuity of
+the metrics time series gathered by Prometheus.
```

Comment:
```suggestion
It is **important** to note that the `job_name` should be a fixed
string (e.g. hardcoded literal). For instance, if you include your charm's
`unit.name`, it would break the continuity of the metrics time series 
gathered by Prometheus when the leader unit changes (e.g. on upgrade
or rescale).
```

Reply:
This suggestion is confusing. `fixed string (e.g. hardcoded literal)` is very verbose compared to the plain `string literal` which is used all over, including the official Python docs.

I'd go with something like:

```
It is **important** to note that `job_name` should be specified in a format which is constant throughout the application lifecycle. A good example would be a hardcoded string, or a string literal which takes formatting only from `charm|application`-level data (e.g. `"foo_{app.name}"`). When a juju application is refreshed, upgraded, or scaled, `unit`-level data may change. A partial list of things to avoid: `unit.name`, charm revision, resource revisions, or any data specific to your job which may change as units are added and removed, upgraded, or relations changed.

> If `job_name` changes for any reason, all existing time series gathered by Prometheus will be orphaned, which will break the continuity monitoring.
```